Xcode/build_mbedtls.sh

```diff
@@ -11,7 +11,7 @@ CMAKE_OPTS="$CMAKE_OPTS \
 
 if [[ "$CONFIGURATION" == "Release*" ]]
 then
-    CMAKE_OPTS="$CMAKE_OPTS -DCMAKE_BUILD_TYPE=RelWithDebugInfo"
+    CMAKE_OPTS="$CMAKE_OPTS -DCMAKE_BUILD_TYPE=MinSizeRel"
```

Comment:
Not sure about this.

Reply:
This was a mistake in code we fixed it separately, Jim suggested to add this minor change with this PR.